airbyte-integrations/connectors/source-mixpanel/source_mixpanel/manifest.yaml

```diff
@@ -0,0 +1,376 @@
+      file_path: "./source_mixpanel/schemas/{{ parameters['name'] }}.json"
+
+  cohort_members_stream:
+    $ref: "#/definitions/engage_stream"
+    $parameters:
+      name: cohort_members
+      path: 2.0/engage
+      field_path: results
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        http_method: POST
+      paginator:
+        $ref: "#/definitions/paginator"
+      partition_router:
+        class_name: "source_mixpanel.components.CohortMembersSubstreamPartitionRouter"
+        parent_stream_configs:
+          - type: ParentStreamConfig
+            stream: "#/definitions/cohorts_stream"
+            parent_key: id
+            partition_field: id
+            request_option:
+              inject_into: body_json
+              type: RequestOption
+              field_name: filter_by_cohort
+    transformations:
+      - class_name: "source_mixpanel.components.EngageTransformation"
+      - type: AddFields
+        fields:
+          - path:
+              - cohort_id
+            value: "{{ stream_partition.get('id') }}"
+      - type: AddFields
+        fields:
+          - path:
+              - browser_version
+            value: "{{ record.browser_version | string }}"
+
+  # No API docs! build based on singer source
+  revenue_stream:
+    $ref: "#/definitions/stream_base"
+    primary_key: "date"
+    $parameters:
+      name: revenue
+      path: 2.0/engage/revenue
+      field_path: results
+    retriever:
+      $ref: "#/definitions/retriever"
+      record_selector:
+        $ref: "#/definitions/selector"
+        extractor:
+          class_name: "source_mixpanel.components.RevenueDpathExtractor"
+          field_path:
+            - "{{ parameters['field_path'] }}"
+    incremental_sync: "#/definitions/incremental_sync"
+
+  # https://developer.mixpanel.com/reference/list-all-annotations-for-project
+  annotations_stream:
+    $ref: "#/definitions/stream_base"
+    $parameters:
+      name: annotations
+      field_path: results
+      path: annotations
+    primary_key: "id"
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        type: CustomRequester
+        class_name: "source_mixpanel.components.AnnotationsHttpRequester"
+        url_base: "https://{{ '' if config.region == 'US' else config.region+'.' }}mixpanel.com/api/"
+        path: "{{ parameters['path'] }}"
+        authenticator: "#/definitions/authenticator"
+        error_handler:
+          $ref: "#/definitions/default_error_handler"
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path:
+            - "{{ 'results' if config.credentials.project_id else 'annotations' }}"
+    incremental_sync:
+      $ref: "#/definitions/incremental_sync"
+      cursor_field: ""
+      # stream is not incremental because date in the record is the date for which annotation was added,
```

Comment:
@midavadim why is there an incremental_sync component if the stream is not incremental?

Reply:
this part is removed, from_date/to_date params are not required by API.